tests-beta/ios/MASVS-STORAGE/MASTG-TEST-0302.md

```diff
@@ -0,0 +1,41 @@
+---
+platform: ios
+title: Sensitive Data Unencrypted in Private Storage Files
+id: MASTG-TEST-0302
+type: [dynamic, filesystem]
+prerequisites:
+- identify-sensitive-data
+profiles: [L2]
+weakness: MASWE-0006
+best-practices: [MASTG-BEST-00xx]
+---
+
+## Overview
+
+This test is designed to complement @MASTG-TEST-0x52-2. Instead of monitoring APIs during execution, it performs a differential analysis of the app's private storage (@MASTG-KNOW-0108) by comparing snapshots taken before and after exercising the app. It also enumerates Keychain items created or modified during the session.
```

Comment:
It's MASTG-TEST-0301, right?
```suggestion
This test is designed to complement @MASTG-TEST-0301. Instead of monitoring APIs during execution, it performs a differential analysis of the app's private storage (@MASTG-KNOW-0108) by comparing snapshots taken before and after exercising the app. It also enumerates Keychain items created or modified during the session.
```